### ISSUE TITLE
Portail des copines

### ISSUE THREAD

**Issue by Fil** [Fil/rezo-squelettes#7]
Il faudrait refaire les visuels qui ont intégré en texte/image le mot COPAINS

**Comment by ghost** [comment]
Je vais tenter le truc sur le logo. Après, je n'ai jamais «gité», donc une fois que j'aurai modifié le logo, je ne sais pas trop comment on fait, ou même où je peux trouver la doc pour apprendre à faire ce qui semble s'appeler un pull requests.

**Comment by Fil** [comment]
Tu peux aussi l’envoyer sur la liste!

Le lun. 23 juil. 2018 à 18:58, Monolecte <notifications@github.com> a
écrit :

**Comment by ghost** [comment]
Oui, mais je pourrais aussi apprendre, ça me servira pour quand je 
serais plus grande…

Le Monolecte
Le blog des agitateurs du vide
http://blog.monolecte.fr

Le 23/07/2018 à 19:35, Philippe Rivière a écrit :

**Comment by ghost** [comment]
Bon, sans les sources, j'ai fait un peu du bricolage, tu me diras si ça te va.
![logorezo2](https://user-images.githubusercontent.com/28186535/43254576-02d5b296-90c8-11e8-9525-c4755cf27ada.png)
![logorezo-ombre2](https://user-images.githubusercontent.com/28186535/43254585-06725d0a-90c8-11e8-807d-4e179f313680.png)
